src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  status: string;
+  handleClick: (event: React.MouseEvent) => void;
+  deleteTodos: (todos: number[], isInUpdate: boolean) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  status,
+  handleClick,
+  deleteTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todos?.filter(td => td.completed !== true).length} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${status === 'all' ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={event => {
+            handleClick(event);
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={`filter__link ${status === 'active' ? 'selected' : ''}`}
+          data-cy="FilterLinkActive"
+          onClick={event => {
+            handleClick(event);
+          }}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={`filter__link ${status === 'completed' ? 'selected' : ''}`}
+          data-cy="FilterLinkCompleted"
+          onClick={event => {
+            handleClick(event);
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method